src/devices/DCMotor/DCMotor.cs

```diff
@@ -69,16 +69,16 @@ protected virtual void Dispose(bool disposing)
         }
 
         /// <summary>
-        /// Creates <see cref="DCMotor"/> instance which allows to control speed in one direction.
+        /// Creates <see cref="DCMotor"/> instance with 3pin driver using only one pin which allows to control speed in one direction.
```

Comment:
`3-pin` or `3 pin` but I'd refrain assuming someone is using driver as they could just use transistor or connect directly for small motors (`using only one pin` is fine)

Reply:
While writing this I was just thinking about the L298N driver that was used in the samples.